packages/vuetify/src/components/VSlideGroup/VSlideGroup.tsx

```diff
@@ -122,97 +167,44 @@ export const VSlideGroup = genericComponent<VSlideGroupSlots>()({
       })
     }
 
-    const disableTransition = ref(false)
-
-    let startTouch = 0
-    let startOffset = 0
+    const isFocused = ref(false)
 
-    function onTouchstart (e: TouchEvent) {
-      const sizeProperty = isHorizontal.value ? 'clientX' : 'clientY'
-      const sign = isRtl.value && isHorizontal.value ? -1 : 1
-      startOffset = sign * scrollOffset.value
-      startTouch = e.touches[0][sizeProperty]
-      disableTransition.value = true
+    function getScrollSize (element?: HTMLElement) {
+      const key = isHorizontal.value ? 'scrollWidth' : 'scrollHeight'
+      return element?.[key] || 0
     }
 
-    function onTouchmove (e: TouchEvent) {
-      if (!isOverflowing.value) return
-
-      const sizeProperty = isHorizontal.value ? 'clientX' : 'clientY'
-      const sign = isRtl.value && isHorizontal.value ? -1 : 1
-      scrollOffset.value = sign * (startOffset + startTouch - e.touches[0][sizeProperty])
+    function getScrollPosition (element?: HTMLElement) {
+      const key = isHorizontal.value ? 'scrollLeft' : 'scrollTop'
+      return element?.[key] || 0
     }
 
-    function onTouchend (e: TouchEvent) {
-      const maxScrollOffset = contentSize.value - containerSize.value
-
-      if (scrollOffset.value < 0 || !isOverflowing.value) {
-        scrollOffset.value = 0
-      } else if (scrollOffset.value >= maxScrollOffset) {
-        scrollOffset.value = maxScrollOffset
-      }
+    function getOffsetSize (element?: HTMLElement) {
+      const key = isHorizontal.value ? 'offsetWidth' : 'offsetHeight'
+      return element?.[key] || 0
+    }
 
-      disableTransition.value = false
+    function getOffsetPosition (element?: HTMLElement) {
+      const key = isHorizontal.value ? 'offsetLeft' : 'offsetTop'
+      return element?.[key] || 0
     }
 
-    function onScroll () {
-      if (!containerRef.value) return
+    function onScroll (e: UIEvent) {
+      const { scrollTop, scrollLeft } = e.target as HTMLElement
 
-      containerRef.value[isHorizontal.value ? 'scrollLeft' : 'scrollTop'] = 0
+      scrollOffset.value = isHorizontal.value ? scrollLeft : scrollTop
     }
 
-    const isFocused = ref(false)
     function onFocusin (e: FocusEvent) {
       isFocused.value = true
-
-      if (!isOverflowing.value || !contentRef.value) return
-
-      // Focused element is likely to be the root of an item, so a
-      // breadth-first search will probably find it in the first iteration
-      for (const el of e.composedPath()) {
-        for (const item of contentRef.value.children) {
-          if (item === el) {
-            scrollOffset.value = calculateUpdatedOffset({
-              selectedElement: item as HTMLElement,
-              containerSize: containerSize.value,
-              contentSize: contentSize.value,
-              isRtl: isRtl.value,
-              currentScrollOffset: scrollOffset.value,
-              isHorizontal: isHorizontal.value,
-            })
-            return
-          }
-        }
-      }
```

Comment:
Should keep this to smoothly scroll to the focused element

Reply:
It works by calling "scrollToChildren" function in necessary cases, like navigation by arrows. Anyway, i rolled it back just in case

Reply:
:+1: This is for tab key focus of arbitrary children in VSlideGroup